### ISSUE TITLE
Not a clue how to build this and all artifacts are expired

### ISSUE THREAD

**Issue by DensestBalloon** [headshot2017/gzdoom-sm64#15]
I never used cmake and have been having issues trying to build this and all the artifacts in actions are expired.

**Comment by headshot2017** [comment]
i've generated [new artifacts](https://github.com/headshot2017/gzdoom-sm64/actions/runs/6124489561), wait for them to finish up and then download it

**Comment by adiee5** [comment]
> i've generated [new artifacts](https://github.com/headshot2017/gzdoom-sm64/actions/runs/6124489561), wait for them to finish up and then download it

yeah, you probably should come up with something better, unless you want to constantly check if artifacts have expired again...
